api/lua/pinnacle/tag.lua

```diff
@@ -194,6 +194,37 @@ function tag.remove(tags)
     end
 end
 
+---Move existing tags to the specified output.
+---
+---#### Example
+---```lua
+---local tag_to_move = Tag.get("1")
+---Tag.switch_output(Output.get_by_name("HDMI-1"), tag_to_move)
+---```
+---
+---@param output pinnacle.output.OutputHandle The output to add tags to.
+---@param ... string The names of the new tags.
+---
+---@overload fun(output: pinnacle.output.OutputHandle, tag_names: string[])
+function tag.switch_output(output, ...)
+    local tag_names = { ... }
+    if type(tag_names[1]) == "table" then
+        tag_names = tag_names[1] --[=[@as string[]]=]
+    end
+
+    local response, err = client:pinnacle_tag_v1_TagService_SwitchOutput({
+        output_name = output.name,
+        tag_names = tag_names,
+    })
+
+    if err then
+        log.error(err)
+        return {}
+    end
+
+    assert(response)
```

Comment:
Assert will crash the config if there is no response, but the response isn't used. I think it would be best to remove this line.